backend/curriculum_tracking/models.py

```diff
@@ -761,6 +761,10 @@ def reviewer_user_names(self):
     def agile_card_status(self):
         return self.agile_card.status
 
+    @property
```

Comment:
Why are these properties? 

KISS, please, Raymond. Please now. We know you are clever. Stop trying to wow us with your fanciness. It's nice that you know about properties, but how is that appropriate here?

Also think about naming.

Here are the appropriate names for a property versus a normal instance method. 

```
topic_progress.duration
topic_progress.get_duration()
```

Once you get to a point where you are able to do simple things we can ask you to do harder things. 

Junior coders think fancy code makes them fancy people. Senior coders know that the simplest code is the best code. 

Please make some perfectly normal super simple functions that just do the thing they are supposed to do without extra bells and whistles. I would rewrite this the moment I try to use it.